src/simple_treeview.rs

```diff
@@ -0,0 +1,90 @@
+//! # TreeView Sample
+//!
+//! This sample demonstrates how to create a TreeView with either a ListStore or TreeStore.
+
+extern crate glib;
+extern crate gtk;
+
+use gtk::prelude::*;
+use gtk::{
+    CellRendererText, Label, TreeStore, TreeView, TreeViewColumn,
+    Window, WindowPosition, WindowType
+};
+
+fn append_column(tree: &TreeView, id: i32) {
+    let column = TreeViewColumn::new();
+    let cell = CellRendererText::new();
+
+    column.pack_start(&cell, true);
+    // we create the column at the position `id`
+    column.add_attribute(&cell, "text", id);
+    tree.append_column(&column);
+}
+
+fn main() {
+    if gtk::init().is_err() {
+        println!("Failed to initialize GTK.");
+        return;
+    }
+
+    let window = Window::new(WindowType::Toplevel);
+
+    window.set_title("Simple TreeView example");
+    window.set_position(WindowPosition::Center);
+
+    window.connect_delete_event(|_, _| {
+        gtk::main_quit();
+        Inhibit(false)
+    });
+
+    // we create a vertical layout to place both tree view and label in the window
+    let vertical_layout = gtk::Box::new(gtk::Orientation::Vertical, 0);
+
+    // we create the label, but because of the closure, we need to wrap it inside
+    // a Rc<Cell<>> to be able to use it outside the closure
+    let label = Label::new(None);
+    let label_copy = label.clone();
+
+    // we create the tree view
+    let tree = TreeView::new();
+    // we create a model with two rows
+    let store = TreeStore::new(&[String::static_type(), i32::static_type()]);
```

Comment:
Is there a reason the numbers column is the second one, has a signed type and starts with 0?